kyverno.tf

```diff
@@ -0,0 +1,107 @@
+locals {
+  kyverno = merge(
+    local.helm_defaults,
+    {
+      name                   = "kyverno"
+      namespace              = "kyverno"
+      chart                  = "kyverno"
+      repository             = "https://kyverno.github.io/kyverno/"
+      create_ns              = false
+      enabled                = false
+      chart_version          = "1.3.3"
+      default_network_policy = true
+    },
+    var.kyverno
+  )
+
+  values_kyverno = <<VALUES
+podSecurityStandard: restricted
```

Comment:
Maybe we should leave to default ? What do you think ?

Reply:
We could switch back to the `default` profile, as the `restricted` one does not seem to be covering common use-cases but more advanced ones (seccomp, no run as-root, ...)
I'll switch to default

Reply:
Alright I think the defaults are fine to get started